static/templates/widgets/todo_widget_tasks.hbs

```diff
@@ -1,19 +1,24 @@
 <br />
 {{#each pending_tasks}}
     <li>
-        <label class="checkbox">
-            <input type="checkbox" class="task" data-key="{{ key }}" />
-            <span></span>
-            <strong>{{ task }}</strong>{{#if desc }} - {{ desc }}{{/if}}
+        <label class="checkbox" id="todo-checkbox">
```

Comment:
You should use a class rather than an ID for an element like this that can appear multiple times in a page.

Reply:
Or better, you can avoid changing the template entirely and just add CSS for `.todo-widget .checkbox`